DEPS

```diff
@@ -18,7 +18,7 @@ deps = {
   },
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@800f9d40b7409239ff192e0be634764e747c7a75",
   "vendor/gn-project-generators": "https://github.com/brave/gn-project-generators.git@b76e14b162aa0ce40f11920ec94bfc12da29e5d0",
-  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@0c281cbdba6fe9c9bbe1d96095e20f857268ddbf",
+  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@3411b967850b23156eea6bb4e08b63ed83833533",
```

Comment:
note: this is NOT up to date with the main branch on WDP because https://github.com/brave/web-discovery-project/commit/a12630c89f1681a5e46ea869885aac3a431591e5#diff-ae8269a7716848b0ecf75491e78f69ade48db1a78b0346d33409adab57116aa9 seems to be causing a build error. 

instead it is the head of https://github.com/brave/web-discovery-project/tree/fix/babel-2, which is branched off of https://github.com/brave/web-discovery-project@0c281cbdba6fe9c9bbe1d96095e20f857268ddbf (the current WDP commit in DEPS)

Reply:
updated